cfgov/regulations3k/jinja2/regulations3k/regulations-listing.html

```diff
@@ -26,6 +26,18 @@ <h4>{{ value.heading or 'Table of contents' }}</h4>
             <a href="{{ regulation.url }}">
                 <span class="title-num">{{ regulation.title }}</span>
             </a>
+            {% if regulation.regulation.future_versions %}
+            <ul>
+            {% for version in regulation.regulation.future_versions %}
+                <li>New amendment effective 
+                    <a href="{{ regulation.url }}/{{ version.effective_date|string }}/">
```

Comment:
This is producing a bad URL with a double slash after the part, like `/policy-compliance/rulemaking/regulations/1026//2018-12-30/`